k0459866/Assignments/Misc/prancercise.py

```diff
@@ -0,0 +1,12 @@
+from sys import argv
+
+# script, text, surround = argv
+
+def prancercise(text, surround):
+    new = text.replace("_", " ")
+    return new.upper()
+    
+# prancercise(text, surround)
+transformed = prancercise("say_hi_to", "-")
```

Comment:
I'd like to see the prancercise function return the new string and have that string get used by assignment06 and assignment07